src/pages/das-api/display-options.md

```
@@ -0,0 +1,95 @@
+
+- [Get Asset](/das-api/methods/get-asset)
+- [Get Assets](/das-api/methods/get-assets)
+- [Get Assets By Owner](/das-api/methods/get-assets-by-owner)
+- [Get Assets By Creator](/das-api/methods/get-assets-by-creator)
+- [Get Assets By Authority](/das-api/methods/get-assets-by-authority)
+- [Get Assets By Group](/das-api/methods/get-assets-by-group)
+- [Search Assets](/das-api/methods/search-assets)
+
+## Performance Considerations
+
+Enabling display options may increase response size and processing time. Only enable the options you need for your specific use case to optimize performance. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)